src/index.js

```diff
@@ -1,39 +1,40 @@
 /**
  * @ngdoc overview
- * @name ng-ovh-apiv7
+ * @name ng-ovh-api-wrappers
  * @description
- * # ng-ovh-apiv7
+ * # ng-ovh-api-wrappers
  *
- * The ng-ovh-apiv7 module is an angular component designed to configure
- * {@link ng-ovh-apiv7.Apiv7Endpoint Apiv7Endpoints}
+ * The ng-ovh-api-wrappers module is an angular component designed to configure
+ * {@link ng-ovh-api-wrappers.ApiEndpoint Apiv7Endpoints}
  * with the same interface as a $resource yet allow for extended configuration by providing
- * {@link ng-ovh-apiv7.Apiv7Request Apiv7Requests} objects that can be modified with chained methods
- * to define the parameters sent to APIv7.
+ * {@link ng-ovh-api-wrappers.ApiRequest Apiv7Requests}
+ *  objects that can be modified with chained methods to define the parameters sent to APIv7.
  */
 
 import angular from 'angular';
 import 'angular-resource';
-import { APIV7_FILTER_COMPARATOR, APIV7_SORT_ORDER } from './constants';
-import { APIV7_ENDPOINT_DEFAULT_ACTIONS } from './endpoint.constants';
-import Apiv7EndpointFactory from './endpoint.factory';
-import Apiv7RequestFactory from './request.factory';
-import service from './service';
-import AggregationResponseTransformerService from './aggregation-response-transformer.service';
-import Apiv7RequestUpgraderService from './request-upgrader.service';
+import { API_ENDPOINT_DEFAULT_ACTIONS } from './common/endpoint.constants';
+import ApiEndpointFactory from './common/endpoint.factory';
+import ApiRequestFactory from './common/request.factory';
+import icebergService from './iceberg/service';
+import v7Service from './v7/service';
+import AggregationResponseTransformerService from './v7/aggregation-response-transformer.service';
+import Apiv7RequestUpgraderService from './v7/request-upgrader.service';
+import ApiIcebergRequestUpgraderService from './iceberg/request-upgrader.service';
 
-const moduleName = 'ngOvhApiv7';
+const moduleName = 'ngOvhApiWrappers';
 
 angular
   .module(moduleName, [
     'ngResource',
   ])
-  .constant('APIV7_FILTER_COMPARATOR', APIV7_FILTER_COMPARATOR)
-  .constant('APIV7_SORT_ORDER', APIV7_SORT_ORDER)
-  .constant('APIV7_ENDPOINT_DEFAULT_ACTIONS', APIV7_ENDPOINT_DEFAULT_ACTIONS)
-  .factory('Apiv7Endpoint', Apiv7EndpointFactory)
-  .factory('Apiv7Request', Apiv7RequestFactory)
-  .service('apiv7', service)
+  .constant('API_ENDPOINT_DEFAULT_ACTIONS', API_ENDPOINT_DEFAULT_ACTIONS)
+  .factory('ApiEndpoint', ApiEndpointFactory)
+  .factory('ApiRequest', ApiRequestFactory)
+  .service('apiv7', v7Service)
```

Comment:
Since we will introduce a new breaking change (the previous one was about using the shared rollup configuration) I encourage you to name all existing services with a `TileCase`.

For instance:
```suggestion
  .service('Apiv7', v7Service)
```

Reply:
Why ? Our convention has always been to use camel case for instances and tile case only for class names, and there `apiv7` is an instance, not a class name

Reply:
Convention changed during the migration from the previous Web Universe Components. But regarding your feedback it seems that convention changed once again to stick with your proposal.

Meaning that we have to update some modules from the Monorepo as well.

Thanks.

@marie-j  sorry for the noise. Please do not pay attention my previous comments.

Reply:
Here is a little sum up of  angular.service definitions:
- ovh-manager-web: 43 Services (6 in camelCase)
- ovh-manager-dedicated: 90 Services (10 in camelCase)
- ovh-manger-cloud: 136 Services (1 in camelCase)
- ovh-manager-telecom: 21 Services (5 in camelCase)
- ovh-api-services: 943 Services (943 in TitleCase)
- manager (monorepo): 19 Services (19 TitleCase)

### Some resources:

A popular resource also use the `TitleCase`:
- From Johnpapa snippets: https://github.com/johnpapa/angular-styleguide/blob/master/a1/assets/sublime-angular-snippets/angular.service.sublime-snippet#L7

- From the Angular CLI as well:
https://github.com/angular/angular-cli/blob/36eba0c9fcf17305a7ef99096c860ad271c5aaf7/packages/schematics/angular/service/files/__name%40dasherize%40if-flat__/__name%40dasherize__.service.spec.ts.template#L3
https://github.com/angular/angular-cli/blob/0070ea46e6eaa25ad49d17c24fe7bed546dc1ec0/packages/angular_devkit/core/src/utils/strings.ts#L73-L89

My proposal is to stick with the `TitleCase`.

Does it make sense for you?

Thanks.

Reply:
I believe your arguments are not relevant:
- the legacy code base is a mess and this isn't new. No convention was enforced and if you think we should follow the convention of the people who wrote `$scope.$parent.$parent.$parent.$parent.$parent.checkIfDomainIsAvailable()` then we have a whole another matter 😄 

- John Papa uses camel case in his most updated guidelines: https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#style-y040

- The Angular CLI handles Angular 2+, not AngularJS, the AngularJS documentation uses camel case: https://docs.angularjs.org/guide/module

And service instances are meant to be used in variables, they are classes, so this looks weird:
```js
this.MyService.doThisThing();
this.myVariable = 3;
```
and it makes it harder to differenciate with
```js
MyService.doThisStaticThing();
```

Note that I believe consistency is more important than conventions so I don't care, but we should create a document, **available to everyone on GitHub**, to note down our conventions :)

Reply:
Both **consistency** and **conventions** are important and have to be put at the same level.

Bear in mind that AngularJS entered a 3 year [Long Term Support](https://docs.angularjs.org/misc/version-support-status) period and it is one of the reason that I've also taking in account what the Angular CLI provide.

Reply:
For now, here are our written rules that we tried to follow so far. These were written at the time by @blaryjp: heart:.
The guys who wrote `$scope.$parent.$parent.$parent.$parent.$parent.checkIfDomainIsAvailable()` didn't have any naming convention ;-)

|Element|Naming style|Example|File name|File name example|usage|
|--- |--- |--- |--- |--- |--- |
|Modules|lowerCamelCase|managerApp|name.js|managerApp.js||
|Controllers|UpperCamelCase + 'Ctrl'|AdminCtrl|name.controller.js|admin.controller.js||
|Directives|lowerCamelCase|userInfos|name.directive.js|user-infos.directive.js||
|Filters|lowerCamelCase|user|name.filter.js|user.filter.js||
|Services / Factories / Providers|lowerCamelCase|user|name.service.js|user.service.js|not a constructor|
|Factories / Providers|UpperCamelCase|Car|name.service.js|car.service.js|constructor|
|Constants|UPPERCASE|USERS_ROLES|moduleName.constants.js|managerApp.constants.js||
|Configuration|non-applicable||moduleName.config.js|managerApp.config.js|.config, .run|

If we still follow these rules, they should be written somewhere publicly on GitHub, I agree with that.

So if we take a look at the table, we should use `apiv7` and I think we should continue to follow these rules as long as we still use angularJS :-) If rollup configuration introduce some new naming convention, we should be first aware of that and then it should also be written somewhere :-) But... do we have resources for writing our own coding style or do we follown an already existing one? This is an other topic :-)

Reply:
As the current written rules are to use `camelCase` in this case and given that we are using AngularJS, I'll stick with those.
If we decide and write that our convention is to use `TitleCase` I can always open another PR taking care of this changes 😃